### ISSUE TITLE
"returning" is broken if there is custom attribute serialization

### ISSUE THREAD

**Issue by timanovsky** [zdennis/activerecord-import#659]
**Repro**

1. Create a model with custom serialization attr, e.g. 
```ruby
class Model < ApplicationRecord 
  serialize :closure, HashWithIndifferentAccess
end
```

2. Regular `import` works fine: `Model.import([Model.new( closure: {a:2}.with_indifferent_access )])`
3. This fails: 
```
> Model.import([Model.new( closure: {a:2}.with_indifferent_access )], returning: ["closure"])
ActiveRecord::SerializationTypeMismatch: Attribute was supposed to be a ActiveSupport::HashWithIndifferentAccess, but was a String. -- "--- !ruby/hash:ActiveSupport::HashWithIndifferentAccess\na: 2\n"
```

Unfortunately I'm not that familiar with AR serialisation flow to propose a fix. 

PS. Rails 4.2 and Postgres

**Comment by jkowens** [comment]
The error is probably happening here:

https://github.com/zdennis/activerecord-import/blob/master/lib/activerecord-import/import.rb#L844-L852

This is where the returning values are set on the model instance. My guess is that the attribute needs to be deserialized before being set.

**Comment by jkowens** [comment]
This has been fixed in the 1.0.5 release.
